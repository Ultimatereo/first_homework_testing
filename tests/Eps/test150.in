4
1.47e-05 4.56e-05 3.85e-05 1.6e-05 2.37e-05
1e-06 7e-06 2.59e-05 3.55e-05 1.07e-05
3.17e-05 5.7e-06 8e-07 8.5e-06 1.18e-05
1.65e-05 1.88e-05 1.77e-05 1.32e-05 3.98e-05
