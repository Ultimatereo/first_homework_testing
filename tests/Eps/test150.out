10.7393
-28.2928
39.764
-23.4331
